src/models/receipt.ts

```diff
@@ -17,7 +17,9 @@ export class Receipt {
   }
 
   get amountFormatted(): string {
-    const value = this.receipt.total_amount ?? this.receipt.net_amount;
+    const value = (
+      (this.receipt.total_amount ?? this.receipt.net_amount) as number
```

Comment:
Two questions:
1. `total_amount` is not marked as optional in the model. Is there a reason you're doing null coalescing here?
2. Both `total_amount` and `net_amount` are both `number` types in the model. How come you're doing `as number`?

Reply:
ok updated!